en/docs/references/ballerina-expression-syntax.md

```diff
@@ -68,6 +88,50 @@ To initialize a variable of type `var` in Choreo, you can use one of the followi
 "Hello!"
 ```
 
+### error
+
+The `error` type supports error values that include a string message. The expressions to initialize this variable type needs to be entered in the `error("<ERROR_STRING>")` format.
```

Comment:
```suggestion
The `error` type represents error values. Error values contain a message of type `string` and optionally a `cause` and additional details about the error. The expressions to initialize a variable of type `error` needs to be entered in the `error("<MESSAGE>")` format.
```

Shouldn't we also include examples of passing the cause and/or details? Aren't these supported yet?